### DISCUSSION TITLE
Soru işaretli yere gelecek şekil?

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#5] [Zeka Sorusu]
![s003](https://github.com/sonsuzus/sonsuzus/assets/52765796/2f8c3bc4-8445-4ec5-9676-037e780d2256)

Soru işaretli yere hangi şekil gelmeli?

**Comment by HabilCcabbar** [comment]
Örüntü:
[Örüntü.txt](https://github.com/sonsuzus/sonsuzus/files/11951917/Oruntu.txt)
Bu örüntüye göre şekil 4 olur. (Boş kare ve artı) (Çizgileri sansürlü olarak varsaydım.)

**Comment by devrimsuzer** [comment]
4. olandır.Örüntüye satır yerine sütundan bakarsak alttaki büyük şekillerden ikisi boş diğeri doludur.Ortadaki sütunun dolu bir şekli var bu yüzden dolu kareleri eledim.Geriye 2. ve 4. kaldı.Satırdan bakarsak artı şekli eksikti.
